data/workspace/src/main/java/com/seugi/data/workspace/model/WorkspaceModel.kt

```diff
@@ -0,0 +1,11 @@
+package com.seugi.data.workspace.model
+
+data class WorkspaceModel(
+    val workspaceId: String,
+    val workspaceName: String,
+    val workspaceImageUrl: String,
+    val workspaceAdmin: Long,
+    val middleAdmin: List<Long>,
+    val teacher: List<Long>,
+    val student: List<Long>,
```

Comment:
```suggestion
    val middleAdmin: ImmutableList<Long>,
    val teacher: ImmutableList<Long>,
    val student: ImmutableList<Long>,
```
Model들은 직접적으로 컴포즈 UI에서 참조하기에 stable한 ImmutableList를 사용해주세요.